trunk/src/gx_head/engine/gx_record.cc

```diff
@@ -367,7 +373,7 @@ void SCapture::compute_static_st(int count, float *input0, float *input1, float
 
 int SCapture::register_par(const ParamReg& reg)
 {
-    static const value_pair fformat_values[] = {{"wav"},{"ogg"},{"w64"},{0}};
+    static const value_pair fformat_values[] = {{"wav"},{"ogg"},{"w64"},{"flac"},{0}};
     if (channel == 1) {
 	reg.registerFloatVar("recorder.file","","S",N_("select file format"),&fformat, 0.0, 0.0, 2.0, 1.0, fformat_values);
```

Comment:
Nice catch. But didn't we need to change the max value for fformat to 3.0 in order to allow to select flac?